src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -1552,6 +1596,8 @@ void populateLoweringONNXElementwiseOpPattern(RewritePatternSet &patterns,
       typeConverter, ctx, enableSIMD);
   patterns.insert<ONNXElementwiseBinaryOpLowering<mlir::ONNXPReluOp>>(
       typeConverter, ctx, enableSIMD, /*isUniBroadcasting=*/true);
+  patterns.insert<ONNXElementwiseUnaryOpLowering<mlir::ONNXClipOp>>(
+      typeConverter, ctx, /*enableSIMD=*/false, /*numScalarArgs=*/2);
```

Comment:
It looks like this is not the way to turn off SIMD for an operation. As I understand we use `using SimdEnabled = NoSimdScalarOp;` in ScalarOp template. @AlexandreEichenberger is this correct?

Reply:
Thanks. This part is removed.